test/micro/org/openjdk/bench/java/lang/ArrayCopyObject.java

```diff
@@ -61,7 +61,7 @@ public MyClass(int val) {
  }
 }
 
-@State(Scope.Benchmark)
+@State(Scope.Thread)
```

Comment:
Are you sure it makes sense as in `main()` method we set `fork(1)` so there's only one thread running the benchmark

Reply:
AFAIK fork value specifies number of times harness should [fork](https://javadoc.io/doc/org.openjdk.jmh/jmh-core/0.6/org/openjdk/jmh/annotations/Fork.html). Also the change is setting scope to thread level not controlling the number of threads.